piksi_multi_rtk_gps/startup/startup.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+  <node name="master_discovery" pkg="master_discovery_fkie" type="master_discovery" >
+  </node>
+
+  <node name="master_sync" pkg="master_sync_fkie" type="master_sync" output="screen">
```

Comment:
Same as above. `master_sync_fkie` is not a dependency of this package and I'd rather not include something that is not strictly needed by the driver.